### ISSUE TITLE
[CoE Starter Kit - BUG] Compliance mails for requested only sent once

### ISSUE THREAD

**Issue by johnravas** [microsoft/coe-starter-kit#4314]
### Describe the issue

I am setting up the compliance "missing details" notification for a single environment, so I can test the experience - before opening it up to all of my makers. I have configured the environment variables to support a simple example in a single environment. When the Admin | Compliance Detail request v3 flow runs, it runs successfully, but I don't receive any kind of notification.

I originally posted about this in the Discussions area so there are more details [here](https://github.com/microsoft/coe-starter-kit/discussions/4211)

When I review the run of the flow, it shows that the environment is being processed, it shows that it sees the app that I should receive notification for  and it sees that there are 22 sessions in the last 30 days (I set the threshold to 5). In looking deeper in the flow these actions are not running:
![image](https://user-images.githubusercontent.com/59970962/204859132-35f01306-6fdf-4036-95ff-c66761ee53d5.png)
![image](https://user-images.githubusercontent.com/59970962/204859249-94f83e81-487f-46c9-9ea0-8d569cef5a00.png)
![image](https://user-images.githubusercontent.com/59970962/204860834-0ee13aa7-1229-40bd-82e8-307f0e905a29.png)

During our discussion conversation, we established that the supporting flows were not enabled.  They are now and did not run at all this week.

Let me know what additional info I can provide.

### Expected Behavior

Expect to receive notification that I need to supply more details on this app

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

3.53

### What app or flow are you having the issue with?

Admin | Compliance details request v3

AB#1076

**Comment by Jenefer-Monroe** [comment]
That failure is caught, its because we sometimes get the session count another way, for you we correctly get it as 22. 
The next thing to check is to look into the condition **_Condition: Audit apps that are broadly shared or admin has requested audit_**
Can you open that and see which way it goes? 
![image](https://user-images.githubusercontent.com/89584709/204945281-40adf9e9-089e-4c85-9874-9d0b2c562d41.png)

**Comment by johnravas** [comment]
Here is where the goes after that action:
![image](https://user-images.githubusercontent.com/59970962/205159218-57f6a93f-7f1b-42aa-b083-f1d93771a45e.png)

Let me know if you need more information.

**Comment by johnravas** [comment]
Hi Jenefer - here is a screenshot of the flow leading up to that action:
![image](https://user-images.githubusercontent.com/59970962/206025825-d09f9c14-36bd-487b-8e94-eb9b725377a1.png)
Let me know if you need anything else.

**Comment by Jenefer-Monroe** [comment]
Yes thats what I thought, so basically once it updates the object to "requested", then it wont do it again, and therefore never send another mail. 
Thanks for investigating. Marking as a bug to address. 

If you want to force out mails for everyone, you would want to unmark all the apps currently marked as Requested, then on the next run, now that the email flow is on, you should get the mails. 
![image](https://user-images.githubusercontent.com/89584709/206043257-ce60aceb-4e43-4f0e-9d6f-f79322272cc4.png)

**Comment by johnravas** [comment]
Thanks for the info Jenefer.  Here is the current state of the app:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/59970962/206304866-ce9fbb04-8bb0-4421-be65-6280eea7b233.png">
The thing is I never received even the initial email for this app.  I can clear this field and force a run of the flow to see if that generates even the first email.   Does that make sense?

**Comment by johnravas** [comment]
Ah yes.  Thanks for the reminder on that Jenefer.  Going to clear the field and test it out.  Thanks for the assistance.

**Comment by johnravas** [comment]
Tested and message received!!  Thanks for the assistance.

**Comment by Jenefer-Monroe** [comment]
great, we'll keep this open as a bug, have tagged it up for triage.

**Comment by johnravas** [comment]
Jenefer - looks like the instructions listed in the Developer Compliance Center app for updating the App Description are out of date.  They indicate:
![image](https://user-images.githubusercontent.com/59970962/206477211-8147d9d3-8e0a-48dd-b616-7c59c068e8c3.png)
It should really indicate something like Edit the app, click on the Settings (gear) icon.  Under the General tab, complete the Description field

We can provide training for our users on this different.  Just thought you would want to know.  I am sure this sequence was correct when the app was initially built.

**Comment by Jenefer-Monroe** [comment]
Thank you, would you mind posting a separate bug for this? Then I will tag that to the same resource to fix up with the other bugs.

**Comment by johnravas** [comment]
Will do.  Thanks Jenefer!!

**Comment by AmarSaiRam** [comment]
We haven’t heard back in a while, so we’ll proceed to close this issue for now. If the issue persists or you need further assistance, feel free to log a new issue. We’re happy to help anytime. Thanks again for your engagement!

**Comment by AmarSaiRam** [comment]
Dear User, 

We noticed that this issue has been pending for quite some time, and we have not received any recent updates from your end. In the absence of further communication, we are proceeding to close this ticket, assuming that your concerns have been addressed or resolved. 
If you continue to experience any difficulties related to this matter, please feel free to open a new support ticket. This will help us ensure prompt and effective assistance, rather than reopening the new Issue. 
Thank you for your understanding and cooperation. 

Best Regards, 
CoE Starter Kit Team
